src/splitInteger.test.js

```diff
@@ -1,21 +1,26 @@
+/* eslint-disable */
 'use strict';
 
 const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  const result = splitInteger(10, 5);
+  expect(result).toEqual([2, 2, 2, 2, 2]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  const result = splitInteger(1, 1);
+  expect(result).toEqual([1]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  const result = splitInteger(10, 3);
+  expect(result).toEqual([3, 3, 4]);
```

Comment:
The expected result seems incorrect. If the parts are sorted in ascending order, the expected result should be [3, 3, 4]. Please verify the logic in the `splitInteger` function to ensure it returns the parts in ascending order.